pyiceberg/catalog/rest.py

```diff
@@ -755,9 +750,12 @@ def _commit_table(self, table_request: CommitTableRequest) -> CommitTableRespons
             CommitFailedException: Requirement not met, or a conflict with a concurrent commit.
             CommitStateUnknownException: Failed due to an internal exception on the side of the catalog.
         """
+        identifier = self._identifier_to_tuple_without_catalog(table.identifier)
+        table_identifier = TableIdentifier(namespace=identifier[0:-1], name=identifier[-1])
```

Comment:
```suggestion
        table_identifier = TableIdentifier(namespace=identifier[:-1], name=identifier[-1])
```
nit: more pythonic and also mirrors

https://github.com/apache/iceberg-python/blob/e4c1748fee220076f04e35ab2f182dd51ca20705/pyiceberg/table/__init__.py#L1678